### PR TITLE
Added LDAPS Variable to settings.py and appropriate changes to basic_…

### PR DESCRIPTION
**Pull request by cisco92**
As requested.
Two new variables in settings.py , appropriate change in basic_ldap.py

### CODE REVIEW

**Comment by wagnst** [comment]
LGTM

File: alerta/settings.py [hunk 1]
```diff
@@ -101,6 +101,10 @@
 LDAP_DOMAINS_BASEDN = {}  # type: Dict[str, str]
 LDAP_ALLOW_SELF_SIGNED_CERT = False
 
+# LDAPS
+LDAPS = False #LDAPS True/False
+LDAPS_CERT = '' #Path to certificate
```
Comment (satterly) [inline alerta/settings.py:106]: Default the `LDAPS_CERT` to `None` and remove the `LDAPS` option .
```
LDAPS_CERT = None  # Path to certificate
```

Then in `basic_ldap.py` it becomes...

```
    if current_app.config['LDAPS_CERT']:
        ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_HARD)
        ldap.set_option(ldap.OPT_X_TLS_CACERTFILE, current_app.config['LDAPS_CERT'])
```

**Comment by KlavsKlavsen** [comment]
Its the autopep8 step that fails.. an automaticly pep8 fix script that fails - and shows no reason why, isn't particularly helpful :( https://pypi.org/project/autopep8/ is the tool used I assume.

**Comment by KlavsKlavsen** [comment]
Seeing as this has stalled - I took the liberty of doing PR 1323 with the suggested adjustments.

**Comment by satterly** [comment]
Closing in favour of #1323. Thanks for your contribution.

**Pull request closed without merging**